uninstall.sh

```diff
@@ -54,4 +54,6 @@ fi
 clear
 echo -e "$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-" #multicolor
 echo -e "$GREEN✓ Fertig: $NORMAL MySpeed wurde deinstalliert."
-echo -e "$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-$GREEN-$NORMAL-" #multicolor
\ No newline at end of file
+echo -e "$NORMAL Du kannst jederzeit MySpeed wieder installieren. Die ANleitung dazu findest du auf sierra-dev.de/myspeed."
```

Comment:
```suggestion
echo -e "$NORMAL Du kannst MySpeed jederzeit wieder installieren. Die Anleitung dazu findest du auf myspeed.gnmyt.dev/setup/linux."
```
Da hat sich ein kleiner Fehler im Wort "Anleitung" aufgetan. ;)
Ich würde allerdings gerne als Installationsanleitung die offizielle Dokumentation dazu verlinken, da dort noch ein bisschen mehr erklärt ist